javascript/app-framework/src/types.ts

```diff
@@ -140,7 +140,12 @@ export interface GeneratedExtraRoutes {
 	children?: ResourceRoute[]
 }
 
-export enum ResourceGroup {
+export enum ResourceGroupType {
```

Comment:
It's weird to me to add `Type` to the end of a type

Reply:
It's like starting an interface with `I` to me.

Reply:
Yeah, but once I had a group with actual items in it (ResourceRouteGroup), it seemed weirder that one was an enum and the other was a structure. We can rename it if it bugs you.